content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -39,6 +39,26 @@ further sub-isolations applied.
 
 A Pod is similar to a set of containers with shared namespaces and shared filesystem volumes.
 
+Pods in a Kubernetes cluster are used in two main ways:
+
+* **Pods that run a single container**. The "one-container-per-Pod" model is the
+  most common Kubernetes use case; in this case, you can think of a Pod as a
+  wrapper around a single container; Kubernetes manages Pods rather than managing
+  the containers directly.
+* **Pods that run multiple containers that need to work together**. A Pod can
+  encapsulate an application composed of
+  [multiple co-located containers](#how-pods-manage-multiple-containers) that are
+  tightly coupled and need to share resources. These co-located containers
+  form a single cohesive unit.
+
+  Grouping multiple co-located and co-managed containers in a single Pod is a
+  relatively advanced use case. You should use this pattern only in specific
+  instances in which your containers are tightly coupled.
+
+  You don't need to run multiple containers to provide replication (for resilience
+  or capacity); if you need multiple replicas, see
+  [Workload management](/docs/concepts/workloads/controllers/).
```

Comment:
Is the indentation of these lines as intended, meaning are they supposed to be part of the text within the [unordered list](https://deploy-preview-44798--kubernetes-io-main-staging.netlify.app/docs/concepts/workloads/pods/#what-is-a-pod)?

Reply:
Yep; see https://deploy-preview-44798--kubernetes-io-main-staging.netlify.app/docs/concepts/workloads/pods/#what-is-a-pod and https://deploy-preview-44798--kubernetes-io-main-staging.netlify.app/docs/concepts/workloads/pods/#how-pods-manage-multiple-containers for how I want it to look.